src/commands/basemaps-github/make.cog.github.ts

```diff
@@ -170,10 +171,24 @@ export class MakeCogGithub {
     logger.info({ imagery: this.imagery }, 'GitHub: Get the master TileSet config file');
     const tileSetPath = fsa.joinAll('config', 'tileset', `${filename}.json`);
     const tileSetContent = await gh.getContent(tileSetPath);
-    const tileSet = JSON.parse(tileSetContent) as ConfigTileSetVector;
-    const newTileSet = await this.prepareVectorTileSetConfig(layer, tileSet);
+    let newTileSet;
```

Comment:
this is typed as a any, generally we should not be using `let` if we can avoid it, could you refactor it so the let is not needed?